examples/valid/plots_list.dvc.yaml

```diff
@@ -20,8 +20,20 @@ plots:
       y:
         evaluation/train/plots/roc.json: tpr
         evaluation/test/plots/roc.json: tpr
+  - confusion: # x dict
+      x:
+        dir/actual.csv: actual
+      y:
+        dir/preds.csv: predicted
+      template: confusion - evaluation/importance.png # Plot an image as str.
+  - roc_vs_prc: # multiple x values
+      x:
+        precision_recall.json: recall
+        roc.json: fpr
+      y:
+        precision_recall.json: precision
+        roc.json: tpr
   - evaluation/importance.png: # Plot an image as dict.
-  - evaluation/importance.png # Plot an image as str.
```

Comment:
Let's keep this. If you don't mind, it would be nice to change to a different path.

Reply:
Sorry, mistakenly removed.